### DISCUSSION TITLE
2.15.0

### DISCUSSION THREAD

**Discussion by TheLastProject** [CatimaLoyalty/Android#794] [Releases]
- Fix cropper not using theme colour                                                                                              
- Fix minor theming issues                                                                                                                                                
- Add pure black dark theme for OLED screens                                                                                                                              

<hr /><em>This discussion was created from the release <a href='https://github.com/CatimaLoyalty/Android/releases/tag/v2.15.0'>2.15.0</a>.</em>

**Comment by anrosci** [comment]
Hello, maybe something is still wrong.  The theme color is visible only in the "+" button at the bottom in dark mode, while in light mode the same color is very faded in fact another color results. See for example the screens attached with the blue theme in light and dark mode, they are different colors and the upper band is only highlighted.![Screenshot_20220210-053858.png](https://user-images.githubusercontent.com/63288613/153338737-008558b8-f04a-432e-ac14-c2dd65a98d7a.png)![Screenshot_20220210-053830.png](https://user-images.githubusercontent.com/63288613/153338758-14027c3d-b7e2-437b-8a97-b780cf6224b2.png)

**Reply by TheLastProject** [reply]
I see the colour affecting the same elements in dark as in light mode (the background effect is just not very accentuated). I definitely see it work the same in light and dark theme.

Did you enable "Pure black background for dark theme"? Because if you did, well, you asked Catima to use pure black instead of the theme colour so then that's expected.

**Reply by anrosci** [reply]
yes I used Pure Black Background for dark theme, but even without it the result is the same.  With the dark theme you can see only the key of the chosen color while with the light theme both the key and the top are of a different color and more faded than the one chosen.  If this is the expected result, that's fine mine was just an observation the app remains very beautiful and functional.  good job!!! now we wait for the archiving functionality.  :-)
